src/messaging/ReliableMessageMgr.h

```diff
@@ -220,11 +219,7 @@ class ReliableMessageMgr
      * set this before actually bringing up the stack and having access to a
      * ReliableMessageMgr.
      */
-    static void SetAdditionalMRPBackoffTime(const Optional<System::Clock::Milliseconds64> & additionalTime)
-    {
-        sAdditionalMRPBackoffTime = additionalTime;
-    }
-#endif // CHIP_DEVICE_CONFIG_ENABLE_DYNAMIC_MRP_CONFIG
+    static void SetAdditionalMRPBackoffTime(Optional<System::Clock::Timeout> additionalTime);
```

Comment:
Is there a reason to pass that by value, not reference?

Reply:
I thought the type is small enough to be passed by value, but this doesn't matter in this case. Reverted.